configuration/local/policy.yml

```diff
@@ -14,14 +14,16 @@ logging:
   appenders:
     - type: console
 
-
-infinispan:
-  bindAddress: 
-  initialHosts: 
-  clusterName: 
-  type: standalone
-  expiration: 8h
-  persistenceToFileEnabled: false
+cache:
```

Comment:
A general point but I'm not sure we should call infinispan a cache. We don't use it to increase access speed to a backing store so its more just a 'store' or a 'repository'

Reply:
I see `cache`  has been replaced with `sessionStoreConfiguration`.  👍